setup.py

```diff
@@ -20,17 +20,15 @@
         'Development Status :: 2 - Pre-Alpha',
         "Intended Audience :: Developers",
         "License :: OSI Approved :: MIT License",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
     ],
     description="Serializers for SQLAlchemy models.",
     extras_require=extras_require,
     install_requires=requirements,
     license="MIT license",
     long_description=readme + "\n\n" + history,
     include_package_data=True,
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*",
+    python_requires=">=3.5",
```

Comment:
Please double check this; in your `environment.devenv.yml` you have specified python>=3.6